app/src/main/java/edu/uc/app/parkfinder/RetrofitClientInstance.kt

```diff
@@ -6,7 +6,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://pkgstore.datahub.io"
+    private val BASE_URL = "https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/"
```

Comment:
Made BASE_URL longer in order to make the @GET for retrofit more readable

Reply:
Usually we only keep the base URL (domain) in the RetrofitClientInstance, and apply the full path in the DAO.   Otherwise, we'd need a BASE_URL for every endpoint we call in that domain, which would obviate the purpose of a BASE_URL.  I'd keep this as it were.